invenio_app_ils/circulation/config.py

```diff
@@ -9,13 +9,10 @@
 
 from invenio_circulation.api import Loan
 from invenio_circulation.pidstore.pids import (_LOANID_CONVERTER,
-                                               CIRCULATION_LOAN_FETCHER,
-                                               CIRCULATION_LOAN_MINTER,
                                                CIRCULATION_LOAN_PID_TYPE)
 from invenio_circulation.search.api import LoansSearch
 from invenio_circulation.transitions.transitions import (
-    CreatedToPending, ItemOnLoanToItemOnLoan, ItemOnLoanToItemReturned,
-    ToCancelled, ToItemOnLoan)
+    CreatedToPending, ItemOnLoanToItemReturned, ToCancelled)
```

Comment:
you might have run `isort` here to fix imports

Reply:
I indeed ran `isort .` but note how the imports are different (one group comes from the circulation module while this other comes from the current project).